packages/happo-core/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "happo-core",
+  "version": "5.0.0",
+  "description": "Command-line tool to visually diff JavaScript components",
```

Comment:
yeah i was copy/pasting package.jsons so the description got carried over for each one. I'll clean up